radixdlt-core/radixdlt/src/main/java/com/radixdlt/DispatcherModule.java

```diff
@@ -195,7 +196,7 @@ public void configure() {
 		bind(new TypeLiteral<EventDispatcher<EpochLocalTimeoutOccurrence>>() { }).toProvider(
 			Dispatchers.dispatcherProvider(
 				EpochLocalTimeoutOccurrence.class,
-				(t, a) -> String.format("Timeout{epoch=%s round=%s leader=%s nextLeader=%s count=%s}",
+				(t, a) -> String.format("bft_timout{epoch=%s round=%s leader=%s nextLeader=%s count=%s}",
```

Comment:
It's a typo or all names are intentionally formatted as "xxx_xxxxxx" (i.e. 3 symbols followed by 6 symbols and separated by underscore)?

Reply:
Yup, exactly trying to make the log stream more readable/parsable